chart/operator/Chart.yaml

```diff
@@ -26,9 +26,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.0.16   # x-release-please-version
+version: 0.0.17   # x-release-please-version
```

Comment:
It is not necessary to increment the release in the PR, that's done automagically 🧙🏽

Reply:
ah I've missed that.. reverted to current release version